src/atomate2/vasp/flows/phonons.py

```diff
@@ -0,0 +1,326 @@
+            raise ValueError(
+                "You can only use other kpath schemes with the primitive standard structure"
+            )
+
+        if self.kpath_scheme not in [
+            "seekpath",
+            "hinuma",
+            "setyawan_curtarolo",
+            "latimer_munro",
+            "all",
+        ]:
+            raise ValueError("kpath scheme is not implemented")
+
+        jobs = []
+
+        if self.use_primitive_standard_structure:
+            # These structures are compatible with many
+            # of the kpath algorithms that are used for Materials Project
+            prim_job = structure_to_primitive(structure, self.symprec)
+            jobs.append(prim_job)
+            structure = prim_job.output
+        elif self.use_conventional_standard_structure:
+            # it could be beneficial to use conventional
+            # standard structures to arrive faster at supercells with right
+            # angels
+            conv_job = structure_to_conventional(structure, self.symprec)
+            jobs.append(conv_job)
+            structure = conv_job.output
+
+        # if supercell_matrix is None, supercell size will be determined
+        if supercell_matrix is not None:
+            self.supercell_matrix = None
+        else:
+            supercell_job = get_supercell_size(
+                structure,
+                self.min_length,
+                self.prefer_90_degrees,
+                **self.get_supercell_size_kwargs,
+            )
+            jobs.append(supercell_job)
+            self.supercell_matrix = supercell_job.output
+
+        if self.bulk_relax_maker is not None:
+            # optionally relax the structure
+            bulk = self.bulk_relax_maker.make(structure, prev_vasp_dir=prev_vasp_dir)
+            jobs.append(bulk)
+            structure = bulk.output.structure
+
+        # get a phonon object from phonopy
+        displacements = generate_phonon_displacements(
+            structure=structure,
+            supercell_matrix=self.supercell_matrix,
+            displacement=self.displacement,
+            sym_reduce=self.sym_reduce,
+            symprec=self.symprec,
+            use_standard_primitive=self.use_primitive_standard_structure,
+            kpath_scheme=self.kpath_scheme,
+            code=self.code,
+        )
+        jobs.append(displacements)
+
+        # perform the phonon displacement calculations
+        vasp_displacement_calcs = run_phonon_displacements(
+            displacements=displacements.output,
+            structure=structure,
+            supercell_matrix=self.supercell_matrix,
+            phonon_maker=self.phonon_displacement_maker,
+        )
+        jobs.append(vasp_displacement_calcs)
+
+        # Computation of BORN charges
+        if self.born_maker is not None and (
+            born_manual is None or epsilon_static_manual is None
+        ):
+            born_job = self.born_maker.make(structure)
+            jobs.append(born_job)
```

Comment:
Can you update this to subsume the additional born logic later on in the function. Note this also assumes you rename born_manual and epsilon_static_manual, and are able to remove the full_born parameter:


```suggestion
        if self.born_maker is not None and (
            born_manual is None or epsilon_static_manual is None
        ):
            born_job = self.born_maker.make(structure)
            jobs.append(born_job)
			
            # I am not happy how we currently access "born" charges
            # This is very vasp specific code
            epsilon_static = born_job.output.calcs_reversed[0].output.epsilon_static
            born = born_job.output.calcs_reversed[0].output.outcar["born"]
            born_run_job_dir = born_job.output.dir_name
            born_run_uuid = born_job.output.uuid
        else:
            born_run_job_dir = None
            born_run_uuid = None
```

Reply:
I updated it. Moved the static run after this whole born logic as well.